.circleci/config.yml

```diff
@@ -905,11 +941,67 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  simple-upgrade-test:
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+    resource_class: large
+    parameters:
+      threescale_release_version:
+        type: string
+        default: "2.6"
+    steps:
+      - install-openshift
+      - run:
+          name: Change to migration-test namespace
+          command: |
+            oc new-project migration-test
+      - oc-observe
+      - run:
+          name: Watch deployment configs
+          command: |
+            while true; do oc describe dc; sleep 600; done
+
+          background: true
+      - run:
+          name: Watch imagestreams
+          command: |
+            while true; do oc describe is; sleep 600; done
+
+          background: true
+      - run:
+          name: Deploy 3scale << parameters.threescale_release_version >> from amp-eval template
+          command: |
+            oc create -f https://raw.githubusercontent.com/3scale/3scale-operator/<< parameters.threescale_release_version >>-stable/pkg/3scale/amp/auto-generated-templates/amp/amp-eval.yml
+            oc new-app 3scale-api-management-eval --param WILDCARD_DOMAIN=lvh.me
+            oc wait --for=condition=available --timeout=-1s $(oc get dc --output=name)
+          no_output_timeout: 30m
+      - checkout
+      - run:
+          name: Build system master image
+          command: |
+            oc create -f test/migration/build-system-master.json
+            oc start-build amp-system-master --wait --follow
```

Comment:
Ideally, we'd use nightly builds image, instead of building a new one.

Reply:
Then you could not check if the current PR is breaking upgrade procedure (by enabling manually the migration test)